components/pipeline-service/base/rbac/openshift-pipelines/configmap-viewer.yaml

```diff
@@ -0,0 +1,33 @@
+---
+# Grant access to authenticated users to be able to view
+# openshift-pipelines configMaps. The user case involves
+# service accounts running automated tests being able to
+# discover the custom URL for the Konflux UI
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: configmap-viewer-role
```

Comment:
Nitpick: is it necessary to include the resource kind in the resource name? Any objection to naming the role `configmap-viewer`? It's not the end of the world to leave the kind in the name, just a little verbose, like `type FooStruct struct {}` 😄